pyhecdss/pyhecdss.py

```diff
@@ -232,6 +232,21 @@ def _get_istat_for_zrrtsxd(self, istat):
         elif istat == 5:  msg = msg + "No pathname(s) found"
         elif istat > 9:  msg = msg + "Illegal call to ZRRTS"
         return msg
+    def _respond_to_istat_state(self, istat):
+        if istat == 0:
+            # everything is ok
+            pass
+        elif istat == 1 or istat == 2 or istat == 3:
+            warnings.warn("Some data or data blocks are missing [istat=" + istat + "]", RuntimeWarning)
+        elif istat == 4:
+            warnings.warn("Found file but failed to load any data", RuntimeWarning)
+        elif istat == 5:
+            # should this be an exception?

```

Comment:
Agree, this should be  an exception.
`raise RuntimeError(msg)`

Reply:
I've updated this and pushed the changes to this pull request.